docs/developer/decision-records/2022-08-01-entity-timestamp/README.md

```diff
@@ -0,0 +1,101 @@
+# Add creation and update timestamp
+
+## Decision
+
+All objects in EDC, which can be persisted in a database, should have a `createdTimestamp` timestamp, all objects that
```

Comment:
I would just call it `created` and `lastUpdated`

Reply:
i wanted to avoid touching the already-existing `createdTimestamp`, but if we do, lets call it `createdAt` and `lastUpdatedAt`?

Reply:
In the DSC, we called it `creationDate` and `modificationDate`

Reply:
> i wanted to avoid touching the already-existing `createdTimestamp`, but if we do, lets call it `createdAt` and `lastUpdatedAt`?

fine for me as well. I don't like the term "timestamp" there